Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:8-jdk-alpine
+
+ADD . /app
+
+RUN /app/gradlew build --build-file /app/build.gradle
+
+EXPOSE 8080
+
+WORKDIR /app
+CMD ["/usr/bin/java", "-jar", "./build/libs/httpstub.jar"]
```

Comment:
`ENTRYPOINT` may be better in order to pass some arguments to the httpstub.
See also https://qiita.com/hnakamur/items/afddaa3dbe48ad2b8b5c